src/components/Footer.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { FC } from 'react';
-import styled from '../styling/styled';
 import { Link } from '@reach/router';
+import styled from '../styling/styled';
 import Container from './Container';
+import epf from '../assets/images/epf.png';
```

Comment:
Maybe we can establish some rule to keep track of variables that holds images. Maybe we can add something like epfImage? But it's just a thought.

Reply:
I did it in other places and here I forgot about it. These variables will have '_img' part at the and of the name

Reply:
Yeah, that's good! Can we stick to camel case though? :slightly_smiling_face:

Reply:
yes :)